lib/util.js

```diff
@@ -151,7 +152,26 @@ module.exports.readFilesToArray = function(fileList) {
   var contentArray = [];
   fileList.forEach(function(filePath) {
     try {
-      contentArray.push(fs.readFileSync(filePath));
+      let extension = filePath.split('.').pop();
+      let content = fs.readFileSync(filePath);
+      if(extension == 'p12' || extension == 'pfx'){
+        let p12Der = forge.util.decode64(content.toString('base64'));
+        let p12Asn1 = forge.asn1.fromDer(p12Der);
+        let p12;
+        try{
+          p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, process.env.KEYSTORE_PASSWORD || "");
+        } catch (e) {
+          loggers.bootstrapLogger.warn("ZWED0173W", e.message);
+          p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, "");
+        }
+        let certData = p12.getBags({bagType: forge.pki.oids.certBag});
+        for(let i = 0; i < certData.length; i++){
+          let currentCert = certData[forge.pki.oids.certBag][i];
+          contentArray.push(Buffer.from(forge.pki.certificateToPem(currentCert.cert), 'utf8'));
```

Comment:
cert is fine when its a cert, but what if its a key? and, how are certificate authorities different from plain certs? same thing? i'm not 100% sure on that.